### DISCUSSION TITLE
Typescript won't compile for Nextjs project

### DISCUSSION THREAD

**Discussion by michael-nelms** [bvaughn/react-resizable-panels#380] [Q&A] [answered]
Trying to get some work released but come to find out project won't build (tsc command) due to this project. Getting errors like:
```
Cannot find module '#is-browser' or its corresponding type declarations.
```

<img width="1322" alt="Screenshot 2024-07-29 at 11 14 41 PM" src="https://github.com/user-attachments/assets/3d594743-0381-4f26-90ee-06bf7ed4f3d9">

Running:
- React 18.20
- Nextjs 13.5.6
- Nodejs 18.20.4
- Typescript 5.2.2

Just wondering if anyone else has run into this issue or might know if I'm just missing something really simple.

**Comment by michael-nelms** [comment, marked as answer]
Nvm I figured it out. Maybe this will help someone else.

I was accidentally importing a module like `import { ImperativePanelHandle } from 'react-resizable-panels/src/Panel'`

Switching my tsconfig `moduleResolution` field to `bundler` pointed out where the issue was. Fixed the import and now everything compiles.

**Reply by bvaughn** [reply]
Glad you got it figured out. Thank you for sharing what the problem was. Hopefully it will help others!
